docker/deploy.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
For build and pushing the docker build image to dockerhub. If you want to use this you will need to use your own space or org.